### ISSUE TITLE
[Minor annoyance] Re-opening the app stops other media

### ISSUE THREAD

**Issue by F1nnM** [futo-org/grayjay-android#261]
Steps to reproduce:
1. Start video in grayjay.
2. Pause video.
3. Minimize grayjay.
4. Close picture-in-picture player -> Important for this bug
5. Start playback in another app, for me YouTube Music.
6. Open Grayjay.

Result:
The video opens, shows a loading animation for a moment, and then shows the video, correctly paused, where I left it. But: It also pauses the other playback. Which really isn't a big issue, but I'd like my music to play until I found a video to watch.

**Comment by CosmicIndustries** [comment]
I have the opposite issue where it plays in background but i like it so i can watch two videos at once

**Comment by F1nnM** [comment]
> I have the opposite issue where it plays in background but i like it so i can watch two videos at once

This actually happens to me when playing a video, pausing it, and connecting to android auto.
Then both my music app starts automatically (intended) and Grayjay also starts playing (not intended).
It took a little bit to notice that, because Grayjays volume is significantly lower than YouTube Music on Android Auto.

(This could have been fixed in one of the recent versions, I came across that maybe one or two weeks ago, and haven't tested it with the latest version)
